planner/core/testdata/integration_suite_in.json

```diff
@@ -226,5 +226,12 @@
       "select approx_percentile(a, 10*10) from t",
       "select approx_percentile(a, 50) from t group by b order by b"
     ]
+  },
+  {
+    "name": "TestConvertRangeToPoint",
+    "cases": [
+      "explain select * from t0 where a > 1 and a < 3 order by b limit 2",
+      "explain select * from t1 where a >= 2 and a <= 2 and b = 2 and c > 2"
```

Comment:
Can we add more test cases such as `explain select * from t0 where a >= 2.0 and a <= 2.0 order by b limit 2` which `a` is `float` type.